docs/artifact-plugin.md

```diff
@@ -149,7 +154,9 @@ Your GRPC server must implement these six methods from the `ArtifactService` int
 
 #### Implementation Notes
 
+<!-- TODO: confirm why artifact.configuration is actually `artifact.artifact_location.plugin.configuration` -->
```

Comment:
@Joibel The spec seems a little convoluted - was this an error in the readme or the spec?

Reply:
Discussed live. @Joibel to rejig the spec to remove the `artifact_location`, so reducing the number of levels needed by 1 --> `artifact.plugin.configuration`